frontend/components/tableDisplay/switchTag.tsx

```diff
@@ -0,0 +1,52 @@
+import { Tag } from "antd";
+
+const SwitchTag = (props) => {
+	const { type, value } = props;
+
+	let element: JSX.Element;
+
+// Switch cases are case-sensitive
+// Ensure that all props are taken in lowercase only
```

Comment:
if you want to be safe, you can lower case the type at the start of this function.

Reply:
currently refactoring this.